src/main/java/com/shippingcostestimator/enterprise/dao/ShipmentDAOStub.java

```diff
@@ -17,7 +17,7 @@
     /***
      * Returns a Shipment object with a specified integer ID.
      * @param id
-     * @return
+     * @return  a specific shipment based on shipmentId.
      */
```

Comment:
Added return description JavaDoc. [Ref](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)

Reply:
While you're here, describe the parameters as well, in the @param tag.